IdentityCore/src/oauth2/aad_base/MSIDAADTokenResponse.m

```diff
@@ -39,7 +39,11 @@ - (void)setAdditionalServerInfo:(NSDictionary *)additionalServerInfo
                              MSID_OAUTH2_RESOURCE,
                              MSID_OAUTH2_CLIENT_INFO,
                              MSID_FAMILY_ID,
+#if !EXCLUDE_FROM_MSALCPP
                              MSID_TELEMETRY_KEY_SPE_INFO,
+#else
+                             @"",
```

Comment:
I think you shouldn't have an empty string field since it changes library's behavior.

Reply:
It doesn't affect MSAL CPP. Also, removing this MACRO will cause adding an empty string in MSIDRequestTelemetryConstants

Reply:
I'm not sure what the result of this empty string being here will be. Could you just not add MSID_TELEMETRY_KEY_SPE_INFO to this dictionary if it's MSAL CPP instead?

Reply:
Oh, I see, this is an array!